Project.toml

```diff
@@ -16,6 +16,7 @@ PeriodicTable = "7b2266bf-644c-5ea3-82d8-af4bbd25a884"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 SBML = "e5567a89-2604-4b09-9718-f5f78e97c3bb"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
```

Comment:
we might eventually need compat with all these packages (just saying, nothing to fix now)